.github/workflows/continuous-integration.yml

```diff
@@ -137,8 +139,8 @@ jobs:
             experimental: false
           - operating-system: "ubuntu-20.04"
             php-version: "8.3"
-            dependencies: "highest"
-            experimental: true
```

Comment:
Please add experimental support for PHP 8.4

Reply:
Done